client/src/templates/Challenges/classic/action-row.tsx

```diff
@@ -50,47 +50,46 @@ const ActionRow = ({
   }
 
   return (
-    <div className='action-row'>
-      <div className='tabs-row'>
-        {!isProjectBasedChallenge && (
-          <button
-            aria-expanded={!!showInstructions}
-            onClick={() => togglePane('showInstructions')}
-          >
-            {t('learn.editor-tabs.instructions')}
-          </button>
-        )}
-        <EditorTabs />
-        <div className='panel-display-tabs'>
-          <button
-            aria-expanded={!!showConsole}
-            onClick={() => togglePane('showConsole')}
-          >
-            {t('learn.editor-tabs.console')}
-          </button>
-          {hasNotes && (
-            <button
-              aria-expanded={!!showNotes}
-              onClick={() => togglePane('showNotes')}
-            >
-              {t('learn.editor-tabs.notes')}
-            </button>
-          )}
-          <button
-            aria-expanded={!!showPreviewPane}
-            onClick={() => togglePane('showPreviewPane')}
-          >
-            <span className='sr-only'>{getPreviewBtnsSrText().pane}</span>
-            <span aria-hidden='true'>{t('learn.editor-tabs.preview')}</span>
-          </button>
-          <button
-            aria-expanded={!!showPreviewPortal}
-            onClick={() => togglePane('showPreviewPortal')}
-          >
-            <span className='sr-only'>{getPreviewBtnsSrText().portal}</span>
-            <FontAwesomeIcon icon={faWindowRestore} />
-          </button>
-        </div>
+    <div className='monaco-editor-tabs'>
+      {!isProjectBasedChallenge && (
+        <button
+          aria-expanded={showInstructions}
+          onClick={() => togglePane('showInstructions')}
+        >
+          {t('learn.editor-tabs.instructions')}
+        </button>
+      )}
+      <EditorTabs />
+      <button
+        className='editor-console-button'
+        aria-expanded={showConsole}
+        onClick={() => togglePane('showConsole')}
+      >
+        {t('learn.editor-tabs.console')}
+      </button>
+      {hasNotes && (
+        <button
+          aria-expanded={showNotes}
+          onClick={() => togglePane('showNotes')}
+        >
+          {t('learn.editor-tabs.notes')}
+        </button>
+      )}
+      <div className='preview-buttons-section'>
+        <button
+          aria-expanded={showPreviewPane}
+          onClick={() => togglePane('showPreviewPane')}
+        >
+          <span className='sr-only'>{getPreviewBtnsSrText().pane}</span>
+          <span aria-hidden='true'>{t('learn.editor-tabs.preview')}</span>
+        </button>
+        <button
+          aria-expanded={showPreviewPortal}
+          onClick={() => togglePane('showPreviewPortal')}
+        >
+          <span className='sr-only'>{getPreviewBtnsSrText().portal}</span>
+          <FontAwesomeIcon icon={faWindowRestore} />
+        </button>
```

Comment:
All of this, is removing the `.action-row` and adding `.preview-button-section` with `editor-console-preview` classes.

Just cleaning, and making it easier to change, without using smart CSS that can target more than intended.